code/modules/mob/living/living.dm

```diff
@@ -737,6 +737,13 @@ Thanks.
 	if(T != loc)
 		handle_hookchain(Dir)
 
+	if(istype(client.eye,/turf/simulated/wall))
```

Comment:
This will runtime for every mob/living without a client

Reply:
Well noted, fixed.